ticket_locator/services/settings_services.py

```diff
@@ -0,0 +1,13 @@
+import environ
+
+
+environ.Env.read_env()
```

Comment:
this code should be in settings.py, environment vars are set up for all project, not just for services